File: GestaoHYS/GestaoHYS.API/ClientApp/src/app/_models/PaymentMethods.ts
export class PaymentMethods{
    id: string;
    description: string;
    paymentMethodsKey: string;
  
    constructor(id : string, name: string, key: string){
        this.id = id;
        this.description = name;
        this.paymentMethodsKey = key;
    }
}

File: GestaoHYS/GestaoHYS.Core/Models/PaymentMethods.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace GestaoHYS.Core.Models
{
    public class PaymentMethods
    {
        [Column("id")]
        [JsonProperty("id")]
        public string id { get; set; }

        [Column("paymentMethodsKey")]
        [JsonProperty("paymentMethodsKey")]
        public string paymentMethodsKey { get; set; }

        [Column("description")]
        [JsonProperty("description")]
        public string description { get; set; }
    }
}
